certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/service/validation/GetCertificatePdfRequestValidator.java

```diff
@@ -0,0 +1,21 @@
+package se.inera.intyg.certificateservice.application.certificate.service.validation;
+
+import static se.inera.intyg.certificateservice.application.common.validator.ValidationUtil.validateCertificateId;
+import static se.inera.intyg.certificateservice.application.common.validator.ValidationUtil.validateUnit;
+import static se.inera.intyg.certificateservice.application.common.validator.ValidationUtil.validateUnitExtended;
+import static se.inera.intyg.certificateservice.application.common.validator.ValidationUtil.validateUser;
+
+import org.springframework.stereotype.Component;
+import se.inera.intyg.certificateservice.application.certificate.dto.GetCertificatePdfRequest;
+
+@Component
+public class GetCertificatePdfRequestValidator {
+
+  public void validate(GetCertificatePdfRequest getCertificatePdfRequest, String certificateId) {
```

Comment:
Should we validate that additionalInfo is included or is it optional?

Reply:
Decided to make it non optional and added validation